packages/themes/types/appTheme.ts

```diff
@@ -44,4 +50,10 @@ export interface Theme {
     shadow?: Color;
     shadowInverted?: Color;
   };
+};
+
+declare module "@emotion/react" {
+  export interface Theme extends LocalTheme {
+    _unused: true; // cannot be empty
+  }
```

Comment:
could you explain a bit what's happening in this file?

Reply:
So for the changes here, I referred to the previous PR related to the Emotion theme updates that looked nearly complete, and their solution seemed to work here. 
I do know with the Emotion 11 update the docs state to update this:
```
It’s now easier to provide a type for Theme. Instead of creating custom instances (like before) you can augment the builtin Theme interface like this:

declare module '@emotion/react' {
  export interface Theme {
    primaryColor: string
    secondaryColor: string
  }
}
```

But here we are extending the `LocalTheme` so it's formatted differently. I am assuming the `_unused: true` is due to potential TS issues 🤔 The "cannot be empty" comment included had me not wanting to remove that bit.

Reply:
Here's a link to the other PR - https://github.com/dcos-labs/ui-kit/pull/578/files 
Let me know if you think I should find a different route for these changes instead.